### PR TITLE
Fix statsd jurisdiction codes to include underscores

### PR DESCRIPTION
**Pull request by harry-wood**
Call the `inferred_jurisdiction_code` method which is defined in `CompanyFetcherBot`.

In practice that method is always available (since all bots are company fetcher bots) but in theory OpencBot::CompanyFetcherBot is optional, and in tests e.g. of the reporting helper where we test out statsd, it is not implemented, hence a seperate test within company_fetcher_bot_spec.

### CODE REVIEW

File: spec/lib/company_fetcher_bot_spec.rb [hunk 1]
```diff
@@ -85,6 +85,12 @@ module UsXxCompaniesFetcher
     end
   end
 
+  describe "#statsd_namspace" do
+    it "uses the inferred_jurisdiction_code method defined in this module gives underscores" do
```
Comment (benjessop12) [inline spec/lib/company_fetcher_bot_spec.rb]: logic is fine, but can we reword this to "uses the inferred_jurisdiction_code method defined in this module to produce jurisdiction_code in snake_case format"

Comment (harry-wood) [reply spec/lib/company_fetcher_bot_spec.rb]: Oh yeah. That wasn't even a proper sentence. Going with your suggestion. 🐍 snake_case (done)
